index.Rmd

```diff
@@ -480,28 +481,39 @@ import matplotlib.pyplot at plt
 import Python.functions as funcs
 
 # Load datasets 
-postboxes = gpd.read_file('data/postbox_locations.shp')
+postboxes = gpd.read_file('data/postbox_locations.gpkg')
 # Then the actual commands
 ```
 
 ## Working directory, relative and absolute file paths
-*At the end of the following section you should be able to explain the difference between __root directory, working directory, absolute path, relative path__* 
 
-In the R and Python examples above we load the datasets by indicating the file location from the data directory `"data/postbox_locations.shp"`. This is an example of using **relative** file paths. We don't include the location of the project directory itself (`"C:/Geoscripting/myproject/data/postbox_locations.sph"`). The reason why they are called *relative* is because they are *relative* to the **working directory**, which is the directory *from which the script is called* (`"C:/Geoscripting/myproject"`). By convention, the working directory is the same as the location of the script itself. This means that you can simply assume that whoever runs your script, will run it from the directory that your script is located.
+*At the end of the following section you should be able to explain the difference between the following:* 
+
+* relative path
+* absolute path
+* working directory, 
+* And the following special directories:
+	 * ` . `
+	 * ` .. `
+	 * ` / ` or the **root** directory
+
+In the R and Python examples above we load the datasets by indicating the file location from the data directory `"data/postbox_locations.gpkg"`.  However, you may have many data folders on your computer, for all types of different projects. So how does the system know to look in the correct one?Moreover, if you share your script with a friend the location of their project and data folders will be different than that of your setup.  It would be a nuisance if they had to change all references to these files in their script. To deal with these issues, we use **relative file paths**. 
```

Comment:
Nice, that's a clear explanation. You are missing a space after the "?" though :)